### ISSUE TITLE
♿ Android - Accessibility - Attendees - Link role is not announced to screen reader users

### ISSUE THREAD

**Issue by ommacc** [CMMS-Esmorga/Esmorga-Project#368]
**Original Jira Issue:** [ESM-436](https://corunamobilemakers.atlassian.net/browse/ESM-436)

## Description

**Current result: **The element is visually styled as a link (underlined), but TalkBack does not announce it as a link.
Screen reader users are not informed that the element is interactive.

**Expected result:** TalkBack should announce the element with its role, for example: “Ver asistentes, link”. This ensures users understand the element is actionable. Users relying on screen readers may not realize that the element is interactive, preventing them from accessing the list of attendees.

## Metadata

- **Type:** Bug
- **Status:** Done
- **Original Jira created:** 2026-01-27T12:36:52.986+0100
- **Priority:** Medium
- **Reporter:** QA Team
- **Assignee:** QA Team
- **Labels:** Android

## Custom Fields

- **Component:** App - Frontend
- **Environment:** QA
- **Priority:** 🟢 Very Low
- **Steps to reproduce:** - Enable TalkBack
- Navigate to the event details screen
- Navigate to link “Ver asistentes”
- **Sprint:** KAN Sprint 37
- **Rank:** 0|i00089:esr
- **Development:** {}

## Related Issues
- **relates to:** #133

**Comment by ommacc** [comment]
**Migrated from Jira** by QA Team on 2026-02-20T13:15:13.110+0100

Fixed: [PR](https://github.com/cmm-apps-android/EsmorgaAndroid/pull/200)

**Comment by ommacc** [comment]
**Migrated from Jira** by QA Team on 2026-02-20T17:06:26.478+0100

Fixed and successfully tested with real Android phone with enabled TalkBack.
